Remotion/Web/Core/Res/Themes/NovaViso/HTML/ListMenu.css

```diff
@@ -2,103 +2,53 @@
 
 table.listMenu
 {
-  border: none;
+  width: 100%;
   border-spacing: 0;
-  border-collapse: collapse;
 }
 
 td.listMenuRow
 {
-  padding-left: 0;
-  padding-right: 0;
-  padding-top: 1px;
-  padding-bottom: 1px;
-  vertical-align: middle;
-  white-space: nowrap;
+  display: flex;
+  gap: var(--remotion-themed-padding);
+  margin-bottom: var(--remotion-themed-padding);
 }
 
-span.listMenuItem
+span.listMenuItem a,
+span.listMenuItemDisabled a
 {
-  margin-right: 10pt;
+  display: flex;
+  text-decoration: none;
+  color: var(--color-primary);
+  gap: var(--remotion-themed-padding);
 }
 
 span.listMenuItem a
 {
-  color: #09457E;
-  padding: 2px;
-  cursor: hand;
   cursor: pointer;
 }
 
-span.listMenuItem a:focus,
-span.listMenuItem a:hover
-{
-  color: Black;
-  padding: 1px;
-  border-width: 1px;
-  border: solid 1px #7CB2E6;
-  background-color: #E5F0FC;
-  outline: none;
-}
-
-span.listMenuItem a:active
-{
-  color: Black;
-  padding: 1px;
-  border-width: 1px;
-  border: solid 1px #569CE0;
-  background-color: #C5E0FC;
-}
-
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie span.listMenuItem a
-{
-  position: relative;
-}
-
-/* Workaround for IE-issue with a span inside an anchor tag blocking the :active pseudo class. */
-body.msie span.listMenuItem a::before
-{
-  content: "";
-  position: absolute;
-  left: 0;
-  right: 0;
-  top: 0;
-  bottom: 0;
-}
-
-span.listMenuItemDisabled
-{
-  margin-right: 10pt;
-}
-
 span.listMenuItemDisabled a
 {
-  padding: 2px;
-  color: gray;
+  color: var(--color-disabled-text);
   cursor: default;
 }
 
-span.listMenuItem a,
-span.listMenuItemDisabled a
-{
-  display: inline-block;
-}
-
 span.listMenuItem a img,
 span.listMenuItemDisabled a img
 {
-  vertical-align: middle;
+  height: var(--programm-icon-size);
+  width: var(--programm-icon-size);
 }
 
-span.listMenuItem a span,
```

Comment:
please add a rule for img+span:empty as display:hidden. Otherwise, an image without text looks strange.

Reply:
Where does this behaviour occur?

Reply:
Okay, just realized that the sample is not configured correctly. You have to disable the text-span explicitly if you don't want it. Just leaving it empty isn't working out.  Could you please update the test page instead to set `Style=Icon`?
![image](https://user-images.githubusercontent.com/6972548/131302721-4a4419e1-7f89-48a3-8434-77f3336006b2.png)